src/model/entities/test/device.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { INewsDevice } from '../inews-device'
+import { TelemetricsDevice } from '../telemetrics-device'
+import { mock } from '@typestrong/ts-mockito'
+
+describe('INewsDevice and TelemetricsDevice', () => {
+  it('should have correct properties and methods', () => {
```

Comment:
I don't think this test gives us much value.
But if you want to keep it then you should atleast split it up so you have one for the INews device and one for the Telemetrics device